compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp

```diff
@@ -627,7 +635,7 @@ LogicalResult AccelSerializer::processOperation(scf::ForOp forOp,
   }
 
   for (Operation &op : *forOp.getBody()) {
-    if (failed(processOperation(&op, subScope))) {
+    if (failed(processOperation(&op, scope))) {
```

Comment:
This change shouldnt be needed? subScope should have everything scope has + induction variable.

Reply:
I have to use `scope` to make indentation correct for the contents within the for loop.

Reply:
But scope wont have the induction variable... surprising that the indentation goes off if you use subscope... 
If its just an indentation issue, could you change this to subScope again. Maybe something else is off with the indentation, but thats cosmetic.